ext/RMagick/rmimage.c

```diff
@@ -4238,11 +4238,11 @@ Image_morphology_channel(VALUE self, VALUE channel_v, VALUE method_v, VALUE iter
   Data_Get_Struct(kernel_v, KernelInfo, kernel);
 
   image = rm_check_destroyed(self);
-  GetExceptionInfo(&exception);
+  exception = AcquireExceptionInfo();
```

Comment:
Would you mind adding a new pull request for changing `GetExceptionInfo` to `AcquireExceptionInfo`?

Reply:
OK, I see. I will split the commit.

Reply:
https://github.com/rmagick/rmagick/pull/343